### PR TITLE
Update Theia, MAJA and Sen2Agri

### PR DESCRIPTION
**Pull request by olivierhagolle**
Dear Kersten,
There has been a lot of activity related to Theia, and MAJA these last months. 

As a result, I updated your awesome catalogue of Sentinel resources, regarding Theia description, and added MAJA atmospheric correction, which is now accessible as a software.  I also added a few things on Sen2Agri, to which my lab contributed a lot. 

thanks in advance if you can include them
Cheers,
Olivier

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,10 +92,12 @@ Specific to Copernicus Sentinel data discovery, download and processing.
   - (pre-)process any Sentinel data
   - also available as [docker](https://github.com/edwardpmorris/docker-snap)
 - [**`Sen2Cor`**](http://step.esa.int/main/third-party-plugins-2/sen2cor/)
-  - athmospheric correction of Sentinel-2 data
+  - atmospheric correction of Sentinel-2 data
   - also available as [python package](https://github.com/umwilm/SEN2COR)
+- [** MAJA ** ](https://logiciels.cnes.fr/en/content/maja)
```
Comment (j08lue) [inline README.md]: Should be `**MAJA**`.

File: README.md [hunk 2]
```diff
@@ -92,10 +92,12 @@ Specific to Copernicus Sentinel data discovery, download and processing.
...
+  - multi-temporal cloud screening and atmospheric correction software that uses time series to better detect coulds and correct atmospheric effects on Sentinel-2 data. Also used within Theia production described above. Developped by CNES, CESBIO and DLR.
```
Comment (j08lue) [inline README.md]: Developed

File: README.md (hunk 1 above)
Comment (olivierhagolle) [reply README.md]: Thanks !

File: README.md (hunk 2 above)
Comment (olivierhagolle) [reply README.md]: Sure (I always do that one). Thanks !

**Comment by kr-stn** [comment]
Thank you for the suggestions (and the kind words) Olivier :+1: 

I'll merge it soon, just some questions regarding phrasing:

- how fast exactly does Theiy provide L2A data that you consider it "near real time"? Most NRT systems I heard of deal in the order of minutes to few hours after acquisition (not production of L1C).
- is there a spatial dataset of the areas covered by Theia or is the coverage decided on a case-to-case basis?

Everything else looks great. I hope you don't mind me shortening some descriptions after merge as I like to keep them very brief (as is the spirit of the awesome lists).

**Comment by olivierhagolle** [comment]
Thanks Kersten, of course you can shorten (I tried !)
- our NRT requirement is less than two days after L1C is produced by ESA, even if we often do much better.  I can't find another word, and "NRT (2 days)" is less brief.
- I am not sure I understand the second question, so I won't be brief ;). When we start producing a site, we produce it permanently in NRT and try to come back in time. There is already a link to a page where we show the tiles. To select the tiles, at the beginning we had asked French user community through a call for proposals. The recent extensions are decided by CNES based on political questions.
Your neighbours from DLR will soon process Germany with MAJA, and deliver the data through CODE-DE.

Thanks,
Olivier

**Comment by olivierhagolle** [comment]
Thanks !
I noticed a typo, in MAJA description, "coulds" instead of "clouds" . Sorry, probably my fault.
Olivier

**Comment by kr-stn** [comment]
I'm rephrasing some parts right now and will fix that error as well.

**Pull request merged**